.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+on: [push, pull_request]
+
+env:
+  PY_COLORS: 1
+
+jobs:
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@stable
```

Comment:
things might break, if black decides to change their formatting. Is there a way to fix the version?

Reply:
Ah that's right, already happened on Travis! I think currently we could only pin it to commit hashes, since the Action is versioned in the black repo itself and they only do tags/releases a few times a year (and nothing since they've added the Action).

But if we wait for this (or add a PR upstream) we can start pinning to black versions: https://github.com/psf/black/issues/1939. I will check, I think it might just be a simple addition to their action.yml and Dockerfile.

Reply:
Ahh if they don't have an option, then we can leave it like that for now.